axum-extra/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+authors = ["David Pedersen <david.pdrsn@gmail.com>"]
```

Comment:
By the way, this field hasn't been required for a while now (and `cargo init` no longer sets it), because it can be very annoying to have info that can change such as an email (and name to lesser extent) in a place where it can't be updated like crates.io.

Reply:
Ah yes thats right! I'll remove it here and for the other crates as well.